packages/sign/src/identity/ci.ts

```diff
@@ -38,8 +37,7 @@ export class CIContextProvider implements IdentityProvider {
   // Invoke all registered ProviderFuncs and return the value of whichever one
   // resolves first.
   public async getToken() {
-    return promise
-      .promiseAny(providers.map((getToken) => getToken(this.audience)))
+    return Promise.any(providers.map((getToken) => getToken(this.audience)))
```

Comment:
Now that we're on Node 16 we can swap our custom `promiseAny` function for the `Promise.any` implementation in the standard library.